modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java

```diff
@@ -419,7 +279,11 @@ public CPInstance addCPInstance(
 			boolean overrideSubscriptionInfo, boolean subscriptionEnabled,
 			int subscriptionLength, String subscriptionType,
 			UnicodeProperties subscriptionTypeSettingsUnicodeProperties,
-			long maxSubscriptionCycles, ServiceContext serviceContext)
+			long maxSubscriptionCycles, boolean deliverySubscriptionEnabled,
```

Comment:
is this method to deprecate?

Reply:
I wanted to clean a bit this class. Too many overloads